src/server/app/routes/events.py

```diff
@@ -90,7 +88,7 @@ def purchase(event_id):
         actor = asys.createActor(EventsActor, None, ActorName.EVENTS_ACTOR)
         payload = {
             'event_id': int(event_id),
-            'quantity': request.get_json().get('quantity')
+            'quantity': int(request.get_json().get('quantity'))
```

Comment:
Bitte als Anmerkung in Server-PR in dann hier entfernen (oder warten auf merge).